src/main/java/com/redhat/devtools/intellij/commonuitest/utils/project/CreateCloseUtils.java

```diff
@@ -67,6 +71,8 @@ public static void createNewProject(RemoteRobot remoteRobot, String projectName,
         newProjectFirstPage.setProjectSdkIfAvailable("17");
 
         if (UITestRunner.getIdeaVersionInt() >= 20221) {
+            remoteRobot.find(ComponentFixture.class, byXpath(XPathDefinitions.JBTEXT_FIELD), Duration.ofSeconds(10)).click(); // Click to gain focus on newProjectFirstPage
```

Comment:
What do you think about adding this as a method to a newProjectFirstPage? Something like `getFocus()` or similar. Would it make sense? 

Note: Not a recommendation, just a genuine question..

Reply:
Refactored